README.md

```diff
@@ -5,6 +5,7 @@
 - Python >= 3.9.9
 - pipenv
 - PostgreSQL
+- Redis (for celery)
```

Comment:
Could you specify here which redis version we are using on Heroku and specify it as a minimum version (like >=)